### PR TITLE
Remove some old compat and update dependencies

### PR DESCRIPTION
**Pull request by StorytellerCZ**
While looking for old compatibility pre Meteor 1.0 I have found few here in Blaze and have taken the liberty to remove them. At the same time I have also updated the dependencies for the affected packages. Version bump is debatable.

### CODE REVIEW

**Comment by jankapunkt** [comment]
Looks good to me

**Review by jankapunkt — APPROVED** [review]
Looks good to me

File: packages/spacebars/package.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: 'spacebars',
   summary: "Handlebars-like template language for Meteor",
-  version: '1.1.0',
+  version: '1.2.0',
```
Comment (filipenevola) [inline packages/spacebars/package.js:4]: Why this should be a minor if the others are patch?

**Review by filipenevola — CHANGES_REQUESTED** [review]
As we are removing code that maybe someone is still using I believe a major would be better but at least a minor would be required for all affected pages, no?

File: packages/spacebars/package.js (hunk 1 above)
Comment (StorytellerCZ) [reply packages/spacebars/package.js:4]: Fixing

**Comment by StorytellerCZ** [comment]
@filipenevola That is what needs to be discussed. These are all compatibility workarounds for before 1.0 version. I'm fine with both approaches.

**Comment by jankapunkt** [comment]
I would favor a major version bump if old compat is removed, too. However, I am not sure how big the implications are for packages depending on `templating@1.x` or `blaze@2.x`. Especially those "unmaintained" ones will definitely become incompatible of those deps are declared, right?

**Comment by filipenevola** [comment]
> I would favor a major version bump if old compat is removed, too. However, I am not sure how big the implications are for packages depending on `templating@1.x` or `blaze@2.x`. Especially those "unmaintained" ones will definitely become incompatible of those deps are declared, right?

This is my concern as well but at least a minor bump we should do on all of them.

**Review by filipenevola — CHANGES_REQUESTED** [review]
I still think we should have a minor bump. Why don't you agree?

**Comment by distalx** [comment]
Minor version bump makes sense to me to keep away from accidentally breaking something.

**Comment by StorytellerCZ** [comment]
Then I think we have a consensus for minor version bump due to older packages per @jankapunkt comment.
Or we could make this a basis for the next major version of Blaze and get in there also the modernization work that @harryadel is working on.

**Pull request merged**